### PR TITLE
Remove arb isolation

### PR DESCRIPTION
**Pull request by efecarranza**
# Changelog

Remove debt-ceiling from ARB on Arbitrum v3.

### CODE REVIEW

File: src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+## Motivation
+
+Since the ARB airdrop, April 2023, the Arbitrum ecosystem has experienced exponential like growth and the ARB liquidity has improved significantly.
+
+With the support of the Risk Service providers, this proposal is to disable Isolation Mode and enable ARB to be used in combination with other assets as collateral.
+
+## Specification:
+
+Ticker: ARB
+Contract Adress: [0x912CE59144191C1204E64559FE8253a0e49E6548](https://arbiscan.io/address/0x912CE59144191C1204E64559FE8253a0e49E6548)
```
Comment (brotherlymite) [inline src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md:21]:
```suggestion
Contract Address: [0x912CE59144191C1204E64559FE8253a0e49E6548](https://arbiscan.io/address/0x912CE59144191C1204E64559FE8253a0e49E6548)
```

File: src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/AaveV3Arbitrum_ARBRemoveIsolation_20240315.sol [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
```
Comment (brotherlymite) [inline src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/AaveV3Arbitrum_ARBRemoveIsolation_20240315.sol:5]:
```suggestion
```

File: src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Chainlink Oracle: [0xb2A824043730FE05F3DA2efaFa1CBbe83fa548D6](https://arbiscan.io/address/0x912CE59144191C1204E64559FE8253a0e49E6548)
```
Comment (brotherlymite) [inline src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md:22]:
```suggestion
Chainlink Oracle: [0xb2A824043730FE05F3DA2efaFa1CBbe83fa548D6](https://arbiscan.io/address/0xb2A824043730FE05F3DA2efaFa1CBbe83fa548D6)
```

File: lib/aave-helpers
Comment (brotherlymite) [inline lib/aave-helpers:1]: @efecarranza can you update with main, i think aave-helpers lib should be already on the latest commit

Comment (brotherlymite) [reply lib/aave-helpers:1]: Perhaps is the reason is breaking the compilation build also

File: src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+This publication proposes disabling Isolation Mode for ARB on Aave v3 Arbitrum.
+
+## Motivation
+
+Since the ARB airdrop, April 2023, the Arbitrum ecosystem has experienced exponential like growth and the ARB liquidity has improved significantly.
+
+With the support of the Risk Service providers, this proposal is to disable Isolation Mode and enable ARB to be used in combination with other assets as collateral.
+
+## Specification:
```
Comment (eboadom) [inline src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md:18]: This Specification is completely arbitrary, not saying anything of what the proposal does

Comment (Luigy-Lemon) [reply src/20240315_AaveV3Arbitrum_ARBRemoveIsolation/ARBRemoveIsolation.md:18]: True.
I suppose that's a flaw in the initial proposal itself.

This has already been voted on though, should we change it only here or also in the forum?

**Review by eboadom — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**